modules/Bio/EnsEMBL/Compara/Utils/MasterDatabase.pm

```diff
@@ -786,11 +776,7 @@ sub create_self_wga_mlsss {
         # POLYPLOID is the restriction of the alignment on the homoeologues
         my $pp_method = $compara_dba->get_MethodAdaptor->fetch_by_type('POLYPLOID');
         my $pp_mlss = create_mlss($pp_method, $species_set);
-        # Pairwise MLSSs between the components
-        my $sub_aln_mlsss = create_mlsss_on_pairs($aln_method, $gdb->component_genome_dbs);
-        # Those MLSSs should remain internal
-        $_->{_no_release} = 1 for @$sub_aln_mlsss;
-        return [$self_lastz_mlss, $pp_mlss, @$sub_aln_mlsss];
+        return [$self_lastz_mlss, $pp_mlss];
```

Comment:
We could maybe delete the component-level LastZ mlss_ids from our master database ?

Reply:
With this update I think we could, yes. Do you want me to do it?

Reply:
Yes please

Reply:
Done